docker/Dockerfile.torch-ort.docs

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu
```

Comment:
should we allow this docker image to be executed as root? maybe adding a non-privileged user would be safer if it is going to be executed on a CI

Reply:
This won't be executed on a CI at the moment. It's for building locally until we onboard to pytorch.org.